src/Postnl.php

```diff
@@ -329,6 +337,42 @@ public function currentStatus($barcode)
     }
 
     /**
+     * @param string $postalCode
+     * @param ComplexTypes\ArrayOfCutOffTime $cutOffTimes
+     * @param string $shippingDate
+     * @param int $shippingDuration
+     * @param string[] $options
+     * @param string $allowSundaySorting
+     * @param string $countryCode
+     */
+    public function getDeliveryDate(
+        $postalCode,
+        $cutOffTimes,
+        $shippingDate,
+        $shippingDuration = 1,
+        $options = ['Daytime'],
+        $allowSundaySorting = 'false',
+        $countryCode = 'NL'
+    ) {
+        /** @var ComplexTypes\GetDeliveryDate $GetDeliveryDate */
```

Comment:
Are you talking about the typehinting or about the class in general? 

In case of the latter, the WSDL to PHP tool apparently created a wrapper around the GetDeliveryDate request (to add the Message). I'll update this.